.distignore

```diff
@@ -8,8 +8,13 @@ tests
 
 # Files
 .distignore
+.distinclude
 .gitattributes
 .gitignore
+.npmrc
+cypress.config.js
+set-latest-wp-version.js
+.*
```

Comment:
With the `.*` rule in place, I think we can drop lines 10-14

Reply:
Yea, that works for me, I wasn't sure if we wanted to be explicitly specific or not. I'll update.